hm_pyhelper/miner_param.py

```diff
@@ -26,9 +27,12 @@ def run_gateway_mfr(args):
 
     command = [gateway_mfr_path]
 
-    if is_rockpi():
-        extra_args = ['--path', '/dev/i2c-7']
+    try:
+        extra_args = ['--path', "/dev/" +
+                      get_variant_attribute(os.getenv('VARIANT'), 'ECCBUS')]
         command.extend(extra_args)
+    except (UnknownVariantException, UnknownVariantAttributeException) as e:
+        LOGGER.exception(e)
```

Comment:
Should not swallow the exception. Also recommend a clearer logging message: "Unable to call gateway_mfr because ECC path could not be determined"

Reply:
Actually, if those errors occur, it would just call gateway_mfr without the path parameter (hence default to i2c-1).  This was my intended behavior.  I suppose I could be more explicit with the message and also set `extra_args` to `None`.